.changeset/moody-bugs-begin.md

```diff
@@ -0,0 +1,7 @@
+---
+'@backstage/plugin-permission-backend-module-policy-allow-all': patch
+'@backstage/plugin-permission-backend': patch
+'example-backend-next': patch
```

Comment:
Patch as this is a change to alpha

Reply:
The example backend is private and doesn't need a changeset.

Reply:
Fixed in eb4b1d4